templates/CRM/Admin/Page/MailSettings.hlp

```diff
@@ -0,0 +1,7 @@
+{htxt id="is_original_eml_attached-title"}
+  {ts}Store original email{/ts}
+{/htxt}
+
+{htxt id="is_original_eml_attached"}
+  <p>{ts}If enabled CiviCRM will store a copy of the original email in .eml file format as an attachment on the activity.{/ts}</p>
+{/htxt}
```

Comment:
do you think we should follow the translation guide for templates as detailed here:
https://docs.civicrm.org/dev/en/latest/translation/extensions/#for-translators-translating-strings-on-transifex
?

Reply:
We're already following this, no?